src/osd/osd_types.cc

```diff
@@ -2270,6 +2274,11 @@ void pg_pool_t::decode(ceph::buffer::list::const_iterator& bl)
                  peering_crush_mandatory_member) = *peering_crush_data;
     }
   }
+  if (struct_v >= 32) {
```

Comment:
The corresponding `DECODE_START...()` hasn't been bumped up. Please do not merge, otherwise we'll  run into an undetected merge conflict with https://github.com/ceph/ceph/pull/60159/.